cmd/server/server.go

```diff
@@ -26,14 +27,19 @@ func main() {
 	ctx, stop := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
 	defer stop()
 
-	err := command.RunJobmanagerServer(
+	listener, err := net.Listen("tcp", ":24482")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
nit: Either make sure you defer-Close the listener on errors or explicitly have RunJobmanagerServer take ownership of it. If panic because RunJobmanagerServer failed (around line 44) the listener leaks.